### PR TITLE
invoke create_workflow now availabel

### PR DESCRIPTION
**Pull request by SooLee**

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -248,8 +260,36 @@ def deploy_core(ctx, name, version=None, no_tests=False):
...
+        cmd_replace_cfg = "mv config.yaml# config.yaml"
+        cmd = ';'.join([cmd_mkdir, cmd_copy, cmd_cd, cmd_modify_cfg, cmd_replace_cfg])
+        print(cmd)
+        run(cmd)
+    else:
+        new_name = name
+    with chdir(new_src):
+        aws_lambda.deploy(os.getcwd(), local_package='../..', requirements='../../requirements.txt')
+    # add roles and environment variables
+    role_arn = 'arn:aws:iam::643366669028:role/lambda_full_s3'  # temporarily hardcorded
```
Comment (j1z0) [inline tasks.py]: So the arn is in the config.yaml and will be set via aws_lambda.deploy so don't think you need to set it a second time.

File: tasks.py [hunk 2]
```diff
@@ -248,8 +260,36 @@ def deploy_core(ctx, name, version=None, no_tests=False):
...
+    lambda_update_config = {'FunctionName': new_name, 'Role': role_arn}
```
Comment (j1z0) [inline tasks.py]: for env variables... also aws_lambda will set those as part of the deploy.. the just need to start with LAMBDA_ see here: https://github.com/j1z0/python-lambda/blob/master/aws_lambda/aws_lambda.py#L356

Comment (j1z0) [reply tasks.py]: so I think you can remove the whole update_function call and lets aws_lambda do it for you.

File: tasks.py (hunk 1 above)
Comment (j1z0) [reply tasks.py]: just need to make sure it's in the config.yml that is being copied across.

File: tasks.py (hunk 2 above)
Comment (SooLee) [reply tasks.py]: Thanks Jeremy. Good to know. But this way we can control different env variables for different lambdas. Not sure how useful that might be though..

File: tasks.py (hunk 1 above)
Comment (SooLee) [reply tasks.py]: Good point!

File: tasks.py (hunk 2 above)
Comment (SooLee) [reply tasks.py]: Sorry I just tried it and it didn't work. Let's keep it this way for now and move on. We can come back later if we wanted to.

File: tasks.py (hunk 1 above)
Comment (SooLee) [reply tasks.py]: I removed the part that adds roles

**Pull request merged**